src/table/TableBodyWrapper.jsx

```diff
@@ -71,7 +71,10 @@ const TableBodyWrapper = ({ rootElement, tableData, constraints, selectionsAPI,
                   selState={selState}
                   selDispatch={selDispatch}
                   tabIndex={-1}
-                  onKeyDown={(evt) => handleKeyPress(evt, rootElement, rowIndex + 1, columnIndex)}
+                  onKeyDown={
+                    (evt) => handleKeyPress(evt, rootElement, rowIndex + 1, columnIndex, cell, selState, selDispatch)
+                    // eslint-disable-next-line react/jsx-curly-newline
```

Comment:
I think it wants you to do something like this
```suggestion
                    (evt) =>
                      handleKeyPress(evt, rootElement, rowIndex + 1, columnIndex, cell, selState, selDispatch)
```

Reply:
<img width="1228" alt="Screenshot 2021-03-17 at 12 46 06" src="https://user-images.githubusercontent.com/4471489/111462463-c2172f00-871e-11eb-9759-352d132e6118.png">
it is same. dot not know why

Reply:
I'm not getting it, have you run `yarn` in a while? some linting stuff might be updated. `yarn lint` is not complaining with my suggestion so let's go with that

Reply:
fixed after running yarn. thanks